src/ontology/mondo-edit.obo

```diff
@@ -430957,13 +431218,45 @@ is_a: MONDO:0035561 {source="https://orcid.org/0000-0001-5208-3432", source="htt
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/9449 ! PRNP
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
 
+[Term]
+id: MONDO:0035639
+name: mixed phenotype acute leukemia with t(9;22)(q34.1;q11.2)
+subset: ordo_subtype_of_a_disorder
+synonym: "MPAL with t(9;22)(q34.1;q11.2); BCR-ABL1" EXACT [Orphanet:589534]
+xref: ICD10:C92.7 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:589534"}
+xref: Orphanet:589534 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020743 {source="https://orcid.org/0000-0001-5208-3432"} ! mixed phenotype acute leukemia
+relationship: RO:0004004 http://identifiers.org/hgnc/3765 ! FLT3
+
+[Term]
+id: MONDO:0035642
+name: mixed phenotype acute leukemia with t(v;11q23.3)
+subset: ordo_subtype_of_a_disorder
+synonym: "MPAL with t(v;11q23.3); KMT2A rearranged" EXACT [Orphanet:589595]
+synonym: "MPAL with t(v;11q23.3); MLL rearranged" EXACT [Orphanet:589595]
+xref: ICD10:C92.6 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:589595"}
+xref: NCIT:C82203 {source="MONDO:equivalentTo"}
+xref: Orphanet:589595 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020743 {source="https://orcid.org/0000-0001-5208-3432"} ! mixed phenotype acute leukemia
+relationship: RO:0004004 http://identifiers.org/hgnc/3765 ! FLT3
+
 [Term]
 id: MONDO:0035645
 name: inherited gynecological cancer-predisposing syndrome
 subset: ordo_group_of_disorders
 xref: Orphanet:589746 {source="MONDO:equivalentTo"}
 is_a: MONDO:0015356 {source="Orphanet:589746"} ! hereditary neoplastic syndrome
 
+[Term]
+id: MONDO:0035682
+name: fibrous dysplasia/McCune-Albright syndrome
+subset: ordo_group_of_disorders
+synonym: "FD/MAS spectrum" EXACT [Orphanet:595216]
+synonym: "FD/MAS syndrome" EXACT [Orphanet:595216]
+synonym: "Fibrous dysplasia/McCune-Albright spectrum" EXACT [Orphanet:595216]
+xref: Orphanet:595216 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018230 {source="https://orcid.org/0000-0001-5208-3432"} ! primary bone dysplasia
```

Comment:
It looks like more than bones are affected ("McCune-Albright syndrome (MAS) is an extremely rare disorder that classically affects the bones, skin, and endocrine system. ")

I would choose "syndromic disorder" as a parent.